frontend/src/components/DragAndDrop/DragAndDrop.js

```diff
@@ -0,0 +1,196 @@
+        setFileUploaded(false);
+        setInvalidJson(false);
+      }, 2500);
+    }
+  };
+
+  return (
+    <div>
+      {lines ? (
+        <App lines={lines} />
+      ) : (
+        <div>
+          <NavBar />
+          <div
+            aria-live="polite"
+            aria-atomic="true"
+            style={{
+              display: "flex",
+              justifyContent: "center",
+              marginTop: "10vh",
+            }}
+          >
+            <div
+              className={`bg-danger toast ${showToast ? "show" : ""}`}
+              role="alert"
+              aria-live="assertive"
+              aria-atomic="true"
+              style={{ zIndex: "1" }}
+            >
+              <div className="toast-body d-flex justify-content-between">
+                Something went wrong! Make sure you have uploaded a valid json
+                file.{" "}
+                <button
+                  type="button"
+                  className="ml-2 mb-1 close"
+                  data-dismiss="toast"
+                  aria-label="Close"
+                  onClick={() => setShowToast(false)}
+                >
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+            </div>
+          </div>
+          <div
+            className="card-body d-grid justify-content-center"
+            style={{
+              position: "absolute",
+              left: "50%",
+              right: "50%",
+              top: "12vh",
+            }}
+          >
+            <form
+              className="form-file-upload"
+              onDragEnter={handleDragEnter}
+              onDragOver={handleDragOver}
+              onDragLeave={handleDragLeave}
+              onDrop={handleDrop}
+              onSubmit={(e) => e.preventDefault()}
+            >
+              <input
+                ref={inputRef}
+                type="file"
+                id="input-file-upload"
+                className="d-none"
+                accept=".json,.jsonl"
+                onChange={handleChange}
+              />
+
+              <div
+                id="label-file-upload"
+                className={dragActive ? "drag-active" : ""}
+                style={{ backgroundColor: `${invalidJson ? "#f00b0b39" : ""}` }}
+              >
+                <div className="text-center">
+                  <animated.div style={flyingAnimation}>
+                    <CloudArrowUpFill size={80} />
+                  </animated.div>
+                  <p className="card-text">
+                    Drag and drop the JSON report file generated by Bowtie in
+                    your local repository!
+                  </p>
+                  <button className="btn btn-primary" onClick={onButtonClick}>
+                    Upload report
+                  </button>
+                  <h5
+                    className={`pt-3 text-danger ${
+                      !invalidJson ? "d-none" : ""
```

Comment:
Maybe it'll be more readable if instead of conditionally adding css class, you would conditionally render whole `<h5>` tag?

Reply:
sure!